.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+# this is a config file for gitpod. Try it out at https://gitpod.io/#https://github.com/temporalio/samples-go/
+ports:
+  - port: 5432
+  - port: 7233
+  - port: 8088
+  - port: 9200
+tasks:
+  - name: Run Hello World Worker
+    init: |
+      cd helloworld
+      gp open README.md
+    command: |
+      gp sync-await server-up
+      go run worker/main.go
+  - name: Run Hello World Workflow
+    init: cd helloworld
+    command: |
+      until go run starter/main.go h; do sleep 5; done; clear; # retry until it works
+      sleep 10 # random buffer just to make really sure temporal is live
+      gp sync-done server-up
+      gp await-port 8088 # temporal web
+      gp preview http://localhost:8088 # reload Temporal Web to show new workflow
+    openMode: split-right
+ - name: Temporalite 
+   init: go install github.com/DataDog/temporalite/cmd/temporalite@latest
+   command: temporalite start --namespace default -f my-test.db
+  - name: Temporal Web/tctl
+    command: |
+      docker run -p 8088:8088 -e TEMPORAL_GRPC_ENDPOINT=host.docker.internal:7233 temporalio/web:1.12.0
```

Comment:
this line works on my machine but not in gitpod. v frustrating. hope that @mikenikles may be able to figure it out, otherwise we will have to revert to using docker compose, which isnt the worst thing, just much slower

Reply:
docker compose equivalent, for reference https://github.com/temporalio/samples-typescript/blob/main/.gitpod.yml

Reply:
See #153 with a fix for the yaml indentation.

This line works for me.

```
gitpod /workspace/samples-go $ docker run -p 8088:8088 -e TEMPORAL_GRPC_ENDPOINT=host.docker.internal:7233 temporalio/web:1.12.0
Unable to find image 'temporalio/web:1.12.0' locally
1.12.0: Pulling from temporalio/web
ddad3d7c1e96: Pull complete 
f54058897200: Pull complete 
b9eac74e56b4: Pull complete 
13e85f13e870: Pull complete 
3fc5362fff95: Pull complete 
bf54ecae65e2: Pull complete 
Digest: sha256:d5bc1aa2d4df383510f29e455722a51e6ce3c48aaf9d5f3bb74f1a7383c86122
Status: Downloaded newer image for temporalio/web:1.12.0
[2021-11-10T00:43:33.265Z] will use insecure connection with Temporal server...
temporal-web ssl is not enabled
temporal-web up and listening on port 8088
```

What error(s) do you see?

Reply:
thanks Mike, the error is not in the terminal, but in the webapp trying to communicate with the temporalite binary:

![image](https://user-images.githubusercontent.com/6764957/141158632-44c8cc66-3959-4271-89ce-be9818c9a42e.png)

when i execute these two steps on my m1 mac mini, i do not get this error. this makes me think there is something special about Gitpod's networking that I need to overcome.

Reply:
The DNS issue can be resolved by adding the `--add-host=host.docker.internal:host-gateway` flag to the `docker run` command.

However, this leads to a `14 UNAVAILABLE: failed to connect to all addresses` error. I don't know yet why that is happening.

Reply:
yes, i can replicate this too :( 
![image](https://user-images.githubusercontent.com/6764957/144992214-d5fcd7a7-9b21-4fc6-8265-0644ebddfc2e.png)

i will try to work on this with @jbreiding  sometime this month.. really appreciate your tips!